snap/hooks/configure

```diff
@@ -3,4 +3,10 @@
 # Register config options if unset,
 # so users can see available options by running
 # `sudo snap get dcgm`.
-[ -z "$(snapctl get nv-hostengine-port)" ] && snapctl set nv-hostengine-port=
+if [ -z "$(snapctl get nv-hostengine-port)" ]; then
+    snapctl set nv-hostengine-port=""
+fi
+
+if [ -z "$(snapctl get dcgm-exporter-listen)" ]; then
```

Comment:
Let's try to be consistent with naming of the options
```suggestion
if [ -z "$(snapctl get dcgm-exporter-port)" ]; then
```

Reply:
It actually takes full address rather than port, it will be misleading to call it `port`. It takes ":9400"

Reply:
I see...
It makes more sense to me then change to `dcgm-exporter-address`, but this is non-blocker

Reply:
Hmm is there any use case for providing an ip to the listen address?  Otherwise an alternative is to expose a numeric port config option, and add the leading `:` in the run wrapper.

Reply:
Yes, if you want to bind it to something different from localhost